app/controllers/professor/exams_controller.rb

```diff
@@ -14,19 +14,25 @@ def new
         [c['name'], c['id']]
       end
     end.flatten(1)
-    pp @courses
   end
 
   def create
-    # TODO make sure prof is registered for the course in BN before creating
     exam_params = params.require(:exam).permit(:name, :file, :course_id, :enabled)
+    course_id = exam_params[:course_id]
+    courses = bottlenose_token.get('/api/courses').parsed
```

Comment:
This could fail, and the uncaught exception would lead to a Err500

Reply:
(In general, we need to be really careful about using the `bottlenose_token` in an unguarded manner)

Reply:
Yes, we should probably catch this at the application level and display an error about contacting bottlenose?

Reply:
#88